sympy/stats/sampling/tests/test_sample_continuous_rv.py

```diff
@@ -174,9 +162,9 @@ def test_sample_continuous():
             imported_lib = import_module(lib)
             if imported_lib:
                 s0, s1, s2 = [], [], []
-                s0 = list(sample(Z, numsamples=10, library=lib, seed=0))
-                s1 = list(sample(Z, numsamples=10, library=lib, seed=0))
-                s2 = list(sample(Z, numsamples=10, library=lib, seed=1))
+                s0 = sample(Z, library=lib, seed=0)
+                s1 = sample(Z, library=lib, seed=0)
+                s2 = sample(Z, library=lib, seed=1)
```

Comment:
Maybe this should have `size=10`.

Reply:
+1. I think the changes are in the representation of outputs(earlier iterator, now a container like arrays). So, input to sampling APIs shouldn't be changed at all. They should be the same and tests should work.

Reply:
I removed numsamples because it is being deprecated and set off warnings during testing. I will add size parameter in the tests. Should numsamples be also added in the test?

Reply:
I think `numsamples` mean number of samples and `size` means size of each sample. Now, if `numsamples` is being removed, then `size` should also be removed and `shape` should be added. This single parameter should be capable of generating `(numsamples,) + size` . So, if users now do, `sample(N, numsamples=10, size=(3, 3))`, then from now onwards they will do, `sample(N, shape=(10, 3, 3))`. Anyways, I would suggest to focus only on changing the output of sampling API. Inputs can be changed in future PRs. Too many things in PR makes it difficult to keep track of things.

Reply:
The point is that these tests previously tested 10 outputs from the seeded rng. With the change here only one output is tested.